src/js/UI.js

```diff
@@ -567,24 +567,27 @@ UI.sayLine = function(e) {
     Players.say(e)
 };
 
-UI.escapeHTML = function(s) {
-    return (
-        ("" + s)
+UI.escapeHTML = function(s, autolink) {
```

Comment:
nice catch! I also noticed some dude with a www.gaming.com or something username getting linked in the chat box. I can use the new param in a followup commit or feel free to put it in there now